arduino-ide-extension/src/browser/boards/boards-config.tsx

```diff
@@ -335,17 +335,12 @@ export class BoardsConfig extends React.Component<
       ports = this.state.knownPorts;
     } else {
       ports = this.state.knownPorts.filter((port) => {
-        if (port.protocol === 'serial') {
+        if (port.protocol === 'serial' || port.protocol === 'network') {
```

Comment:
I don't understand why these two specific protocols are treated differently than any other arbitrary protocol. In this new era of pluggable discovery, where no assumptions or restrictions are placed on which communication channel might be used between the Arduino tooling and an Arduino board, it seems best for the Arduino IDE to generalize the handling of ports.

Reply:
I opened a follow-up: https://github.com/arduino/arduino-ide/issues/1334